crates/server/openapi.yaml

```diff
@@ -1026,6 +1015,21 @@ components:
           minLength: 107
           maxLength: 107
           example: "ecdsa-p256:MEUCIQCzWZBW6ux9LecP66Y+hjmLZTP/hZVz7puzlPTXcRT2wwIgQZO7nxP0nugtw18MwHZ26ROFWcJmgCtKOguK031Y1D0="
+    ContentSourceMap:
+      type: object
+      description: The map of content digest to sources.
+      patternProperties:
+        "^sha256:[a-f0-9]{64,64}$":
```

Comment:
This pattern would have to change in future versions that support other algorithms. Is there a better way of generalizing this / do we need to encode the algorithm constraint here at all?

Reply:
Yeah, I think I'll just relax these patterns to be more generic, something like `[a-z0-9-]+:[a-f0-9]+` for `AnyHash` and change signatures similarly (but keep the base64 regex after the colon).

The API spec should be algorithm agnostic.